### ISSUE TITLE
Style guides: set a max width for the page content

### ISSUE THREAD

**Issue by damithc** [nus-cs2103/website#158]
It is better to set a max width for the page. Otherwise the content become hard to read when using a wide screen browser.

**Comment by acjh** [comment]
The current max-width (for the page) is 1000px, the width of content is 900px (90%).

- For screens above 1000px (original upper limit since 9624088aeffac4c92517ceff1634f3a839fe0770, still in use)
body {
    max-width: 1000px;
    ...
    width:90%;
}

- For screens in the range: 501px < max-device-width < 1000px (merged in #132)
    body{
        max-width: 100%;
    }

- For screens 500px and below (proposed for #155, to maximise screen 'real estate')
    body {
        ...
        width: 96%;
    }

**Comment by damithc** [comment]
What is the typical max width that is not too hard to read? The current one looks a bit too wide.
The screenshot is from my desktop.
![image](https://cloud.githubusercontent.com/assets/1673303/12219848/3f325d4e-b78f-11e5-87fa-ce3edaf3b2e1.png)

**Comment by acjh** [comment]
1000px should be fine.

I found that the problem comes from coding-standards-cpp.html (as with the other coding standards) being loaded separately, outside schedule.html. As such, the 'common.css' stylesheet is not linked.

<a href="http://acjh.github.io/website/contents/coding-standards-cpp.html">Here</a> is how it looks like after I linked it. Should I submit a PR together with the other coding standards?
